ghcide/test/src/Development/IDE/Test.hs

```diff
@@ -83,7 +83,7 @@ expectNoMoreDiagnostics timeout =
   expectMessages STextDocumentPublishDiagnostics timeout $ \diagsNot -> do
     let fileUri = diagsNot ^. params . uri
         actual = diagsNot ^. params . diagnostics
-    liftIO $
+    unless (actual == List []) $ liftIO $
```

Comment:
Huh, does this actually happen?

Reply:
Yes, this is actually a 4th failure vector for this test